src/Player.cpp

```diff
@@ -2172,7 +2276,13 @@ void Player::StepStrumHopo( int col, int row, const RageTimer &tm, bool bHeld, b
 
 						bool bBright = m_pPlayerStageStats && m_pPlayerStageStats->m_iCurCombo>(unsigned int)BRIGHT_GHOST_COMBO_THRESHOLD;
 						if( m_pNoteField )
+						{
 							m_pNoteField->DidHoldNote( col, HNS_Held, bBright );
+						}
+						if(m_new_field != nullptr)
+						{
+							m_new_field->did_hold_note(col, HNS_Held, bBright);
```

Comment:
More for future discussion, but can we at least agree on naming standards? I am suspecting going with underscores is the wave of the future, but I'd like it formalized.